cmd/dp-hierarchy-builder/main.go

```diff
@@ -65,6 +66,7 @@ func main() {
 	healthChecker := healthcheck.NewServer(
 		config.BindAddr,
 		config.HealthCheckInterval,
+		config.HealthCheckRecoveryInterval,
```

Comment:
I know it wasn't you, but could you change the variable `config` to be `cfg` to avoid variable/package namespace conflicts

Reply:
happy to! 👍